packages/tools/src/tools/CrosshairsTool.ts

```diff
@@ -469,7 +471,7 @@ class CrosshairsTool extends AnnotationTool {
   }
 
   handleSelectedCallback = (
-    evt: EventTypes.MouseDownEventType,
+    evt: EventTypes.MouseDownEventType | EventTypes.TouchStartEventType,
```

Comment:
I'm not sure I follow. For a lot of tools `MouseDownEventType` is changed to `InteractionEventType` but here it is `MouseDownEventType` and `TouchStartEventType` which actually I prefer !

Reply:
CC @wayfarer3130 what do you prefer?